src/HTMLMesh.js

```diff
@@ -259,8 +286,11 @@ function html2canvas( element ) {
 			y = rect.top - offset.top - 0.5;
 			width = rect.width;
 			height = rect.height;
+			// On Quest the font used to draw on canvas is bigger than on
+			// the desktop, compensate for this.
+			const maxWidth = width * 1.01; // 1.005 is good, but use 1.01 to be sure
```

Comment:
This one is hacky. I didn't have the same result on Meta browser and Chrome desktop.
Also without this, the label for radio weren't in one line but on two lines.

Reply:
I don't love it but whatever works tbh